lib/index.ts

```diff
@@ -1,3 +1,5 @@
 export { CloudConfigFileTypes, MapsDocIdToTree } from './types';
 
 export { issuesToLineNumbers, getTrees, getLineNumber } from './issue-to-line';
+
+export { parseFileContent } from './yaml-parser';
```

Comment:
I _think_ we need to document somewhere, likely in the README and the source, that this function uses a different underlying YAML parser to the `getTrees()` function and that we need to migrate the underlying code in `getTrees()` to also use the 'yaml' library and drop 'yaml-js'.